rest-api/grant_permissions.sql

```diff
@@ -0,0 +1,7 @@
+GRANT SELECT ON ${db_name}.* TO '${READONLY_DB_USER}'@'%';
+GRANT SELECT, INSERT, UPDATE, DELETE, CREATE TEMPORARY TABLES ON ${db_name}.*
+TO '${RDR_DB_USER}'@'%';
+GRANT SELECT, INSERT, UPDATE, DELETE, DROP, ALTER, CREATE, INDEX, REFERENCES,
+CREATE TEMPORARY TABLES, CREATE VIEW ON ${db_name}.* TO '${ALEMBIC_DB_USER}'@
+'%';
```

Comment:
This doesn't feel like it should be on its own line. If you need to wrap, take everything after the "TO" with it.

Reply:
fixed